liiatools/datasets/social_work_workforce/SWFtools/analysis/seniority.py

```diff
@@ -9,6 +9,13 @@
 
 
 def seniority():
+    """
+    Determine seniority codes for each record in the input CSV file.
```

Comment:
I would probably add a line about adding a column for leavers ["LeftOrNot"] and new starters ["NewOrNot"]

Reply:
I have updated these docstrings in my branch here: https://github.com/SocialFinanceDigitalLabs/liia-tools/tree/193-csww-ingest-sc